src/instance/InstanceService.ts

```diff
@@ -839,7 +833,7 @@ export class InstanceService {
     try {
       await account.login(loginData, true, ClientInfo);
     } catch (error) {
-      logger.error(`[${formatDate()}]`, error);
+      console.error(`[${formatDate()}]`, error);
```

Comment:
Why not using the `logger` instance?

Reply:
Because we are using console everywhere else.. There are not many cases though, so should I change everything to use a logger?

Reply:
Yes, please

Reply:
Done!